### ISSUE TITLE
Abandoned ?

### ISSUE THREAD

**Issue by TEDDAC** [snaptoken/lua-tutorial#15]
Is the project abort ?

**Comment by doubleorseven** [comment]
Hi,
Sorry for the late reply.
Yes I'm no longer seeking to work against GCE.
Maybe it will change soon but I guess the project will need a re-write so maybe I'll look into other solutions

**Comment by cokacode** [comment]
Yes sir!, The guy is writing novel now I think, and selling at amazon.it

**Comment by AikoRamalho** [comment]
The project owner died a few years ago.

**Comment by yanjankaf** [comment]
How do you know that ?

**Comment by thacuber2a03** [comment]
I don't think they're affirming, they're probably asking. But still, is it abandoned? Because I would really like to do this tutorial, I'm kind of a Lua fan 😅

**Comment by yanjankaf** [comment]
No, ond person previously said the owner of repo died.

But from what I know, this snaptoken website is maintained by "paige ruten"
and she is the one writing all those tutorials, and she is pretty much
alive on twitter.

Maybe she is just busy.

**Comment by paigeruten** [comment]
I'm alive 🙂 This is a very ambitious project which I have barely started, and haven't found the time to work on in the last few years. But I would still love to do it eventually. If/when I do get back into it, I will try to release it chapter-by-chapter to my mailing list which you can sign up for here: http://eepurl.com/cIOGCD (I have never sent anything to this mailing list, and won't until I have a new tutorial to release)

**Comment by thacuber2a03** [comment]
yippee :D

**Comment by Zireael07** [comment]
Bump. Any progress on the actual tutorial?

**Comment by tokisuno** [comment]
bump, any update? @paigeruten
